### PR TITLE
Add TestFlight scheme with Wormholy

### PR DESCRIPTION
**Pull request by whoiswillma**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ target 'Eatery' do
     pod 'NVActivityIndicatorView'
     pod 'SnapKit'
     pod 'lottie-ios'
+    pod 'Wormholy', :configurations => ['Debug', 'TestFlight']
```
Comment (ywsang) [inline Podfile:26]: what's the difference between the `Debug` and `TestFlight` configurations? based on the `.xcsheme` files in the commit, it looks like the testflight config isn't used?

**Review by ywsang — CHANGES_REQUESTED** [review]
one small question

File: Podfile (hunk 1 above)
Comment (whoiswillma) [reply Podfile:26]: I dug around and found the TestFlight .xcscheme file inside xcuserdata, whereas all the rest are in xcshareddata? I'm not sure what's up with that, but I recreated the scheme and that seemed to have moved it

Comment (whoiswillma) [reply Podfile:26]: I had to create a TestFlight scheme (instead of just using Debug) because FLEX uses UIWebView which Apple no longer accepts

**Review by ywsang — APPROVED** [review]
ah, got it! looks good

**Pull request merged**